src/api/app/controllers/concerns/webui/requests_filter.rb

```diff
@@ -1,8 +1,28 @@
 module Webui::RequestsFilter
   extend ActiveSupport::Concern
 
+  ALLOWED_INVOLVEMENTS = %w[all incoming outgoing].freeze
   TEXT_SEARCH_MAX_RESULTS = 10_000
 
+  def set_filter_involvement
+    @filter_involvement = params[:involvement].presence || 'all'
+    @filter_involvement = 'all' if ALLOWED_INVOLVEMENTS.exclude?(@filter_involvement)
+  end
+
+  def set_filter_state
+    @filter_state = params[:state].presence || []
+    @filter_state = @filter_state.intersection(BsRequest::VALID_REQUEST_STATES.map(&:to_s))
+  end
+
+  def set_filter_action_type
+    @filter_action_type = params[:action_type].presence || []
+    @filter_action_type = @filter_action_type.intersection(BsRequestAction::TYPES)
+  end
+
+  def set_filter_creators
+    @filter_creators = params[:creators].presence || []
+  end
```

Comment:
Looks like those methods are not reused anywhere else, I would just inline them in `index`

Reply:
Just take a look at the description of the last commit:

_This way, they can be reused by other pages with requests filters._

Reply:
Oh, wait, this is going to be reused later for package and project right? Nvm then